build/elasticsearch/bin/es-docker

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Run Elasticsearch and allow setting default settings via env vars
+#
+# e.g. Setting the env var -Ecluster.name=testcluster
+#
+# will cause Elasticsearch to be invoked with -Ecluster.name=testcluster
+#
+# see https://www.elastic.co/guide/en/elasticsearch/reference/5.0/settings.html#_setting_default_settings
+
+es_opts=''
+
+while IFS='='  read -r envvar_key envvar_value
+do
+    case $envvar_key in
+    [a-z.]*)
```

Comment:
Actually, maybe we should play it safe and just set `LC_ALL=C`, (though don't `export` it, because that would be a really mean thing to do to Elasticsearch).